rootfs-configs.yaml

```diff
@@ -0,0 +1,35 @@
+rootfs_configs:
+  buster:
+    rootfs_type: debos
+    debian_release: buster
+    arch_list:
+      - 'amd64'
+      - 'arm64'
+      - 'armel'
+      - 'armhf'
+      - 'i386'
+      - 'mips'
+      - 'mipsel'
+      - 'mips64el'
+    extra_packages_remove:
+      - bash
+      - e2fslibs
+      - e2fsprogs
+      - klibc-utils
+      - libext2fs2
+      - libgnutls30
+      - libklibc
+      - libncursesw6
+      - libp11-kit0
+      - libunistring2
+      - sensible-utils
+    extra_files_remove:
+      - '*networkd*'
+      - '*resolved*'
+      - tar
+      - patch
+      - dir
+      - partx
+      - find
+    script: ""
+    docker_image: godebos/debos
```

Comment:
As mentioned before, please remove references to Docker from this config.  Also, we're not currently using this image in KernelCI but `kernelci/base-debos`.

Reply:
removed